pytests/requirements.txt

```diff
@@ -0,0 +1,3 @@
+redis==5.0.0rc2
```

Comment:
Can you remind me whats the issue with 5.0.1?

Reply:
The tests freeze for me; I presume the TFCALLASYNC freezes. I wasn't into updating the version so much, so I didn't investigate further.

Reply:
Ok, lets leave it for another PR.